src/core/ai_circuit_breaker.py

```
@@ -0,0 +1,53 @@
+"""
+Circuit breaker implementation for AI operations.
+Moved from app/core/ai_circuit_breaker.py
+"""
+
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from typing import Optional, Dict, List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)